pkg/cmd/hash.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/xerrors"
+	api "sigs.k8s.io/k8s-container-image-promoter/pkg/api/files"
+	"sigs.k8s.io/k8s-container-image-promoter/pkg/filepromoter"
+)
+
+// GenerateManifestOptions holds the parameters for a hash-files operation
+type GenerateManifestOptions struct {
+	// BaseDir is the directory containing the files to hash
+	BaseDir string
+}
+
+// PopulateDefaults sets the default values for GenerateManifestOptions
+func (o *GenerateManifestOptions) PopulateDefaults() {
```

Comment:
Stub?

Reply:
Yeah, there currently aren't any fields that need initializing.  But we still want to follow the pattern. 
 Added a comment to make that clearer.